composer.json

```diff
@@ -20,16 +20,49 @@
     ],
     "require": {
         "php": ">=5.3.0",
-        "psr/http-message": "^1.0"
+        "psr/http-message": "^1.0",
+        "raphhh/trex-reflection": "^0.2.3"
```

Comment:
I could do it manually, but this would mean to create a function or class somewhere – or it would not get DRY. I'm not aware of any other way to share code which wouldn't get *public* :unamused: